File: Backend/src/main/java/hadi/springSecurity/models/responses/TokenResponse.java
package hadi.springSecurity.models.responses;

public class TokenResponse
{
	private String accessToken;
	private String refreshToken;
	private String message;
	private boolean success;
	
	public TokenResponse()
	{
	}

	public TokenResponse(String accessToken, String refreshToken, String message, boolean success)
	{
		super();
		this.accessToken = accessToken;
		this.refreshToken = refreshToken;
		this.message = message;
		this.success = success;
	}

	public String getAccessToken()
	{
		return accessToken;
	}

	public void setAccessToken(String accessToken)
	{
		this.accessToken = accessToken;
	}

	public String getRefreshToken()
	{
		return refreshToken;
	}

	public void setRefreshToken(String refreshToken)
	{
		this.refreshToken = refreshToken;
	}

	public String getMessage()
	{
		return message;
	}

	public void setMessage(String message)
	{
		this.message = message;
	}

	public boolean isSuccess()
	{
		return success;
	}

	public void setSuccess(boolean success)
	{
		this.success = success;
	}
	
	
	
}

File: Frontend/src/app/models/responses/TokenResponse.ts
export class TokenResponse
{
    private _accessToken: string;
    private _refreshToken: string;
    private _message: string;
    private _success: boolean;


    constructor(accessToken: string, refreshToken: string, message: string, success: boolean)
    {
        this._accessToken = accessToken;
        this._refreshToken = refreshToken;
        this._message = message;
        this._success = success;
    }

    public get accessToken(): string
    {
        return this._accessToken;
    }

    public get refreshToken(): string
    {
        return this._refreshToken;
    }

    public get message(): string
    {
        return this._message;
    }

    public get success(): boolean
    {
        return this._success;
    }

    public set accessToken(value: string)
    {
        this._accessToken = value;
    }

    public set refreshToken(value: string)
    {
        this._refreshToken = value;
    }

    public set message(value: string)
    {
        this._message = value;
    }

    public set success(value: boolean)
    {
        this._success = value;
    }

}